react/features/base/media/components/web/Audio.js

```diff
@@ -52,6 +55,20 @@ export default class Audio extends AbstractAudio {
         );
     }
 
+    /**
+     * Stops the audio HTML element.
+     *
+     * @returns {void}
+     */
+    stop() {
+        if (this._ref) {
+            this._ref.pause();
+
+            // $FlowFixMe
```

Comment:
I know it's not your change, but I think it's acceptable to require the this._ref to not be null, since the component is rendered without a condition, so then the ref will be surely set. If you remove the question mark from the field declaration then, all the FlowFixMe hints can go away.

Reply:
Well, we have as soon as the component is rendered. My takeaway is that it's very rare not to have a value for it (between component instantiation and component mount) but we have to have flow annotations to workaround that. I just think we could just assume that it always have a value (except that 0.001 seconds) and then we can get rid of the annotations :)